apps/gitness/src/AppMFE.tsx

```diff
@@ -191,7 +191,7 @@ export default function AppMFE({
                                 Link={Link}
                                 NavLink={NavLink}
                                 Outlet={Outlet}
-                                location={window.location}
+                                location={{ ...window.location, state: {}, key: '' }}
```

Comment:
whats the key being used for? @vardanbansal-harness

Reply:
It's not used anywhere right now, had to add it to fix the type issue. In React router v6, `Location` interface now has a required `key` and `state` properties as well.